scripts/bootstrap/github-infra.sh

```
@@ -1,26 +1,341 @@
+        bootstrap::warn "github-infra: label '$name' create failed (continuing with remaining labels)"
+      }
+  done
+}
+
+bootstrap::_invite_reviewers() {
+  local full_repo=$1 reviewers_csv=$2
+  local agent login
+
+  # Split CSV into individual agent names.
+  local IFS=','
+  set -- $reviewers_csv
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)